nextjs/deployment.yaml

```
@@ -0,0 +1,63 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: demo-nextjs
+  name: nextjs
+  namespace: parca
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: demo-nextjs
+  template:
+    metadata:
+      labels:
+        app: demo-nextjs
+    spec:
+      containers:
+      - image: parca-demo:nextjs
+        name: parca-demo-nextjs
+        resources:
+          limits:
+            cpu: '100m'
+            memory: '256Mi'
+        ports:
+        - containerPort: 5000
+          name: http
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    service: demo-nextjs
+  name: demo-nextjs
+  namespace: parca
+spec:
+  type: NodePort
```

nit: I don't think there's a specific reason for this Service to have a NodePort, is there? 
```suggestion
  type: ClusterIP
```